std_msgs/Header header

std_msgs/Int32 robot_id

# completion time
float32 tf

# goal for robot with id <robot_id>
geometry_msgs/Point goal
